sql/atxs/atxs_test.go

```diff
@@ -45,6 +45,7 @@ func TestGet(t *testing.T) {
 	for _, want := range atxList {
 		got, err := atxs.Get(db, want.ID())
 		require.NoError(t, err)
+		want.AtxBlob = types.AtxBlob{}
```

Comment:
Just for clarification: every ATX has a matching `AtxBlob` and for golden ATXs that `AtxBlob` is `[]byte{}` and not `nil`? There is never an ATX without an `AtxBlob`?

Reply:
`AtxBlob` is not a pointer. For checkpointed ATXs it has the "zero value". It was placed inside of  `types.ActivationTx` as a shortcut to make adding new ATXs easier (an alternative is to have something `atxs.Store(db, atx, blob)`, maybe blob could be "optional" like `blob ...byte`. Perhaps, that is a mistake, and `AtxBlob` should not be in `types.ActivationTx` :thinking:. 

This override here is needed because `atxs.Get` doesn't read the blob.

Reply:
OK, I mixed up expected and actual value in the assertion and was of the opinion the value is set in the actual variable rather than the expected one.

I don't think `types.ActivationTx` should contain the raw ATX bytes. Not including them should speed up DB fetches and reduce the memory footprint of code that deals with lots of ATXs. It however is important that any ATX that is added to the DB can only be added by also providing the raw bytes, except in the case of restoring a checkpoint, but for this a `atxs.AddGolden` could serve the purpose of adding a golden ATX that does not have the raw bytes.